.github/workflows/lint.yml

```diff
@@ -18,6 +18,8 @@ jobs:
           go-version: 1.19.x
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.52.2 # TODO: Catching new linter errors now. Need to fix those and upgrade to v1.53.0
```

Comment:
`v1.53.0` was released about 12 hours ago and now it's reporting some false positives and some old missed ones.
https://github.com/flanksource/canary-checker/actions/runs/5153337251/jobs/9280448518?pr=1056